scapy/layers/dns.py

```diff
@@ -128,7 +128,6 @@ def dns_encode(x, check_built=False):
 def DNSgetstr(*args, **kwargs):
     """Legacy function. Deprecated"""
     raise DeprecationWarning("DNSgetstr deprecated. Use dns_get_str instead")
```

Comment:
I think this should have been something like
```python
warnings.warn("DNSgetstr deprecated...",DeprecationWarning)
```

Reply:
Do you want me to add a commit to this PR to make this change?